automatic/keeweb/tools/chocolateyInstall.ps1

```diff
@@ -0,0 +1,18 @@
+﻿$ErrorActionPreference = 'Stop';
+
+$toolsPath = "$(Split-Path -parent $MyInvocation.MyCommand.Definition)"
+
+$packageArgs = @{
+  packageName    = 'keeweb'
+  fileType       = 'exe'
+  file           = "$toolsPath\KeeWeb-1.6.2.win.ia32.exe"
+  file64         = "$toolsPath\KeeWeb-1.6.2.win.x64.exe"
+  softwareName   = 'keeweb*'
+  silentArgs     = '/S'
+  validExitCodes = @(0)
+}
+
+Install-ChocolateyInstallPackage @packageArgs
+
+# Lets remove the installer as there is no more need for it
+Remove-Item -Force -ea 0 "$toolsDir\*.exe","$toolsDir\*.exe.ignore" 
```

Comment:
This can fail. This is what we use, and you can remove ignore files

`ls $toolsPath\*.exe | % { rm $_ -ea 0; if (Test-Path $_) { sc "$_.ignore" '' } }`